library/src/main/java/com/google/android/exoplayer2/mediacodec/MediaCodecRenderer.java

```diff
@@ -202,6 +202,8 @@ private static String buildCustomDiagnosticInfo(int errorCode) {
   private boolean outputStreamEnded;
   private boolean waitingForKeys;
 
+  private long latestResetPosition;
```

Comment:
I don't think you need to make any changes to this class. You should find that buffer.isDecodeOnly() is already true for the buffers you're skipping, which means they'll be skipped anyway. This flag should be being set in DefaultTrackOutput.readData because decodeOnlyUntilUs should be set to the last seek position.

Try reverting this class and seeing if everything still works. If so, great. If not, is something wrong with my reasoning above, or is something broken?